articles/virtual-machines/windows/change-availability-set.md

```diff
@@ -142,12 +142,16 @@ The following script provides an example of gathering the required information,
     Add-AzureRmVMDataDisk -VM $newVM -Name $disk.Name -VhdUri $disk.Vhd.Uri -Caching $disk.Caching -Lun $disk.Lun -CreateOption Attach -DiskSizeInGB $disk.DiskSizeGB
     }
 
-    #Add NIC(s)
-    foreach ($nic in $OriginalVM.NetworkInterfaceIDs) {
-        Add-AzureRmVMNetworkInterface -VM $NewVM -Id $nic
-    }
-
-    #Create the VM
+    #Create new availability set if it does not exist    $availSet = Get-AzureRmAvailabilitySet -ResourceGroupName $rg -Name $newAvailSetName -ErrorAction Ignore    if (-Not $availSet) {    $availset = New-AzureRmAvailabilitySet -ResourceGroupName $rg -Name $newAvailSetName -Location $OriginalVM.Location    }
+    #Create the basic configuration for the replacement VM    $newVM = New-AzureRmVMConfig -VMName $OriginalVM.Name -VMSize $OriginalVM.HardwareProfile.VmSize -AvailabilitySetId $availSet.Id    Set-AzureRmVMOSDisk -VM $NewVM -VhdUri $OriginalVM.StorageProfile.OsDisk.Vhd.Uri  -Name $OriginalVM.Name -CreateOption Attach -Windows
+    #Add Data Disks    foreach ($disk in $OriginalVM.StorageProfile.DataDisks ) {     Add-AzureRmVMDataDisk -VM $newVM -Name $disk.Name -VhdUri $disk.Vhd.Uri -Caching $disk.Caching -Lun $disk.Lun -CreateOption Attach -DiskSizeInGB $disk.DiskSizeGB    }
+    }    foreach ($nic in $OriginalVM.NetworkInterfaceIDs) {        Add-AzureRmVMNetworkInterface -VM $NewVM -Id $nic    }
+
+#Add NIC(s)
+foreach ($nic in $OriginalVM .NetworkProfile.NetworkInterfaces) {

```

Comment:
It looks like there is an extra space between "OriginalVM" and the ".NetworkProfile..."